src/app/shared/comment-editor/comment-editor.component.ts

```diff
@@ -251,11 +255,11 @@ export class CommentEditorComponent implements OnInit {
     return toInsert;
   }
 
-  private insertUploadUrl(filename: string, uploadUrl: string) {
+  private getNewCursorPosition(insertingString: string, insertedString: string) {
```

Comment:
Extremely small nit: pass in `filename` here since the `insertingString` format is the same, and perform string interpolation within this function.

Reply:
The reason I wanted to do this was because I wanted to reuse the `insertingString` variable in `this.getNewCursorPosition` and `this.commentField.value.replace` (around lines 291). 

If I insert a filename, won't I need to have two variables of the same string? Not sure if we want to have a repeat of `[Uploading ${filename}...]` both inside and outside the function but I'll change it and you can take a look if that's what you want

Reply:
@dingyuchen please take a look if you've time, thank you!